pkg/agent/agent.go

```diff
@@ -7,11 +7,19 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	"github.com/spiffe/spire/pkg/common/profiling"
```

Comment:
nit: this should be down further with the other github imports

Reply:
Ouch, I'll fix that